docs/dev/index.md

```diff
@@ -0,0 +1,68 @@
+
+# OpenSearch SQL/PPL Engine Development Manual
+
+## Introduction
+
++ [Architecture](intro-architecture.md): a quick overview of architecture
++ [V2 Engine](intro-v2-engine.md): introduces why we developed new V2 engine
++ Concepts
++ Quickstart
+
+---
+## Clients
+
++ **CLI**
++ **JDBC Driver**
++ **ODBC Driver**
++ **Query Workbench**
```

Comment:
Isn’t this a UI feature ?
can we also have a UI section associated to the SQL ?

Reply:
yes, I was planning to add some brief intro or link to their repo. Added here just to give users a complete picture of our eco system.